app/boards/arm/nrfmicro/nrfmicro_11.dts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2020 Pete Johanson, Okke Formsma, joric
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "arduino_pro_micro_pins.dtsi"
+
+/ {
+	model = "nrfmicro_flipped";
+	compatible = "joric,nrfmicro";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		// zephyr,console = &uart0;
+		//zephyr,bt-mon-uart = &uart0;
+		//zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		blue_led: led_0 {
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+			label = "Blue LED";
+		};
+	};
+
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&uart0 {
```

Comment:
Please remove this. The nano accidentally had this twice, and it's duplicated from the real one below.